app/components/frame-viewer.cjsx

```diff
@@ -26,13 +26,31 @@ module.exports = React.createClass
     playing: false
     playbackRate: 1
     frameDimensions: {}
+    viewBoxDimensions: {
+      x: 0,
+      y: 0,
+      width: 0
+      height: 0
+    }
+    panEnabled: false
+    zooming: false
+    zoomingTimeoutId: 0
```

Comment:
I think the initial state value of `zoomingTimeoutId` is misleading. Instead of `0`, the value should be `null`. `zoomingTimeoutId` refers to the id returned from `setTimeout()`, so I would like to keep the property name, but change the property's initial value. Thoughts?